content/applications/marketing/email_marketing.rst

```diff
@@ -1,440 +1,792 @@
-lists` field drop-down menu.
+.. tip::
+   More than one mailing list can be chosen from the :guilabel:`Select mailing lists` field.
 
-Then, Odoo will only send this email to contacts on that specific mailing list.
+Odoo then sends the email to contacts on that specific mailing list(s).
 
 .. seealso::
-   :doc:`/applications/marketing/email_marketing/mailing_lists`
+   :doc:`email_marketing/mailing_lists`
 
 When the :guilabel:`Recipients` field is clicked, a drop-down menu of other options is revealed.
 Each option provides different ways Odoo can create a target audience for the email.
 
+.. image:: email_marketing/recipients-dropdown.png
+   :align: center
+   :alt: View of recipients drop-down menu in the Odoo Email Marketing application.
+
 Those options (excluding the default :guilabel:`Mailing List`) provide the option to create a more
-specified recipient filter, in an equation-like format.
+specified recipient filter, in an equation-like format, which appears beneath the
+:guilabel:`Recipients` field.
 
-The :guilabel:`Recipients` field options are as follows:
+The :guilabel:`Recipients` field options, other than the default :guilabel:`Mailing List` option,
+are as follows:
 
-- :guilabel:`Applicant`: filter focuses on specific job applicants in the database.
-- :guilabel:`Contact`: filter focuses on specific contacts in the database.
-- :guilabel:`Event Registration`: filter focuses on people in the database who purchased event
+- :guilabel:`Contact`: focuses on specific contacts in the database.
+- :guilabel:`Event Registration`: focuses on people in the database who purchased event
   registrations.
-- :guilabel:`Event Track`: filter focuses on people in the database who hosted a specific talk
-  (track) at an event.
-- :guilabel:`Lead/Opportunity`: filter focuses on leads or opportunities in the database.
-- :guilabel:`Mailing Contact`: filter focuses on specific mailing contacts in the database.
-- :guilabel:`Sales Order`: filter focuses on a specific sales orders in the database.
+- :guilabel:`Lead/Opportunity`: focuses on leads or opportunities in the database.
+- :guilabel:`Mailing Contact`: focuses on specific mailing contacts in the database.
+- :guilabel:`Sales Order`: focuses on a specific sales orders in the database.
 
-If the specified recipient fields don't automatically reveal themselves, simply click the
-:guilabel:`Add Filter` button beneath the :guilabel:`Recipients` field, and Odoo reveals the
-necessary equation fields to further drill down the target recipients for this mailing.
+Add recipient filter
+~~~~~~~~~~~~~~~~~~~~
 
-Add a recipient filter
-----------------------
+To add a more specific recipient filter to any :guilabel:`Recipient` option, select any recipient
+option (other than :guilabel:`Mailing List`), and click the :guilabel:`Modify filter (right-facing
+arrow)` icon beneath the :guilabel:`Recipient` field to reveal three subsequent filter rule fields,
+formatted like an equation.
```

Comment:
The framing of this feature is *technically* correct but it seems to lack real-world application knowledge, specifically:

- It is *highly encouraged* to use detailed targeting criteria for the **Recipients** field. It's not a "nice-to-have" or an optional feature— it's very necessary, and one line of targeting logic typically is not enough to run a smart email campaign.
- **Mailing List** is indeed a fair **Recipients** option to choose from as a base but it's not really the standard (not unless you're buying marketing lists and important them into Odoo, which we'd frown upon anyway). The more standard **Recipient** sources would be **Lead/Opportunity** or **Event Registration**, to which, much more detailed targeting criteria would be added on top of those seed sources.

Here's an example of how detailed our targeting should should be in this section
![Screenshot 2024-02-23 at 12 28 35 PM](https://github.com/odoo/documentation/assets/5496809/6c6f9f89-0559-4269-b57c-0badac868bf7)